todo_app/test_system.py

```diff
@@ -0,0 +1,65 @@
+import os
+import pytest
+import todo_app.app as app
+from todo_app.trello import create_board, delete_board
+from threading import Thread
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import dotenv
+import requests
+ 
+@pytest.fixture(scope='module')
+def test_app():
+
+    #file_path = dotenv.find_dotenv('.env')     
+
+    # Create the new board & set it to env variable
+    board_id = create_board() 
+    os.environ['TRELLO_BOARD_ID'] = board_id
+
+    # Get the new board list ids and update the environment variables
+    params = (
+        ('key', os.environ['TRELLO_KEY']),
+        ('token', os.environ['TRELLO_TOKEN']),
+        ('fields', 'all')
+    )
+    boardid = os.environ['TRELLO_BOARD_ID']
+    r = requests.get('https://api.trello.com/1/boards/' + boardid + '/lists', params=params)
+
+    to_do_id = r.json()[0]['id']
+    doing_id = r.json()[1]['id']
+    done_id = r.json()[2]['id']
+
+    os.environ['TRELLO_TODO_LIST_ID'] = to_do_id
+    os.environ['TRELLO_DOING_LIST_ID'] = doing_id
+    os.environ['TRELLO_DONE_LIST_ID'] = done_id
+
+    # construct the new application   
+    application = app.create_app()   
+
+    # start the app in its own thread.  
+    thread = Thread(target=lambda: application.run(use_reloader=False))  
+    thread.daemon = True  
+    thread.start()   
+    yield app   
+
+    # Tear Down     
+    thread.join(1)  
```

Comment:
Does this line achieve anything (I don't believe we're in a separate calling thread)?